cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py

```diff
@@ -0,0 +1,2650 @@
+import pytest
+
+from cfme import test_requirements
+
+
+@pytest.mark.manual
+@test_requirements.smartstate
+@pytest.mark.tier(1)
+def test_ssa_datastore_files():
+    """
+    Check datastore files are fetched correctly
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: SmartState
+        caseimportance: medium
+        initialEstimate: 1/2h
+        testtype: integration
+    """
+    pass
+
+
+@pytest.mark.manual
+@test_requirements.smartstate
+@pytest.mark.tier(1)
+def test_ssa_vm_users():
+    """
+    Check users are fetched correctly for analysed VM
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: SmartState
+        caseimportance: medium
+        initialEstimate: 1/2h
+        testtype: integration
+    """
+    pass
+
+
+@pytest.mark.manual
+@test_requirements.smartstate
+@pytest.mark.tier(1)
+def test_ssa_datastore_files_unicode():
+    """
+    Make sure https://bugzilla.redhat.com/show_bug.cgi?id=1221149 is fixed
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: SmartState
+        caseimportance: medium
+        initialEstimate: 1/2h
+        startsin: 5.3
+        testtype: integration
+    """
+    pass
+
+
+@pytest.mark.manual
+@test_requirements.smartstate
+@pytest.mark.tier(1)
+def test_ssa_groups_scvmm2k12_windows2012r2_ntfs():
+    """
+    Add SCVMM-2012 provider.
+    Perform SSA on Windows 2012 server R2 VM having NTFS filesystem.
+    Check whether it retrieves Groups.
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: SmartState
+        initialEstimate: 1/2h
+        startsin: 5.6
+    """
+    pass
+
+
+@pytest.mark.manual
+@test_requirements.smartstate
+@pytest.mark.tier(1)
+def test_ssa_groups_azure_ubuntu():
+    """
+    1. Add Azure provider
+    2. Perform SSA on Ubuntu instance.
+    3. Check Groups are retrieved.
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: SmartState
+        initialEstimate: 1/3h
+        startsin: 5.6
+    """
+    pass
+
+
+@pytest.mark.manual
+@test_requirements.smartstate
+@pytest.mark.tier(1)
+def test_ssa_groups_scvmm2k12_centos_xfs():
+    """
+    Add SCVMM-2012 provider.
+    Perform SSA on CentOS VM.
+    Check whether Groups retrieved.
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: SmartState
+        initialEstimate: 1/2h
+        startsin: 5.6
+    """
+    pass
+
+
+@pytest.mark.manual
+@test_requirements.smartstate
+@pytest.mark.tier(1)
+def test_ssa_groups_azure_rhel():
+    """
+    1. Add Azure provider
+    2. Perform SSA on RHEL instance.
+    3. Check Groups  are retrieved.
+
```

Comment:
add ```testSteps``` under polarion tags.

Reply:
I will update it while Automating tests.